tests/loan-token/Administration.test.js

```diff
@@ -50,6 +51,9 @@ contract("LoanTokenAdministration", (accounts) => {
     let sovryn, SUSD, WRBTC, RBTC, BZRX, loanToken, loanTokenWRBTC;
 
     async function deploymentAndInitFixture(_wallets, _provider) {
+        // need to deploy the mutex in the initialization. Otherwise, the global reentrancy prevetion will not be working & throw an error
```

Comment:
```suggestion  
        // Need to deploy the mutex in the initialization. Otherwise, the global reentrancy prevention will not be working & throw an error.
```